source/_posts/2020-07-11-android-18-110-release.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "Home Assistant Companion for Android 1.8 - 1.10"
```

Comment:
1.8 or 1.10? This might be a bit confusing to the reader

Reply:
So we were actually going to talk about 3 different releases here, would it make more sense to list them out instead of the `-`?

Reply:
updated to be more clear now :)